Sources/Logging/Logging.swift

```diff
@@ -39,20 +39,41 @@ import WASILibc
 /// logger.info("Hello World!")
 /// ```
 public struct Logger {
-    @usableFromInline
-    var handler: LogHandler
+    /// A private property to hold the boxed `LogHandler`.
+    private var _handler: Box<LogHandler>
 
     /// An identifier of the creator of this `Logger`.
     public let label: String
+    
+    /// A computed property to access the `LogHandler`.
+    public var handler: LogHandler {
+        get {
+            return _handler.value
+        }
+        set {
+            if !(isKnownUniquelyReferenced(&_handler)) {
+                _handler = Box(value: newValue)
+            } else {
+                _handler.value = newValue
```

Comment:
please use explicit `self.` everywhere, otherwise these things are very hard to read especially in diffs which might not have enough context to spot where this is from.